Podfile

```diff
@@ -472,9 +474,12 @@ post_install do |installer|
     # =====================================
     #
     installer.pods_project.targets.each do |target|
```

Comment:
I didn't work to add [this tweak](https://github.com/WordPress/gutenberg/blob/4d5cdfea44bd974b833de3b2e600b34853c265ea/packages/react-native-editor/ios/Podfile#L28) because I believe this project now only compiles for Xcode 13+.